app/presenters/idv/in_person/verification_results_email_presenter.rb

```diff
@@ -48,7 +48,7 @@ def service_provider_homepage_url
       end
 
       def needs_proof_of_address?
-        !(enrollment.current_address_matches_id || enrollment.capture_secondary_id_enabled)
+        !enrollment.current_address_matches_id
```

Comment:
The same point I made previously applies here. The correct refactor of this method would always return `false`.
```suggestion
        false
```